tests/manager/bug0357.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+PHPC-357: The exception for "invalid namespace" does not list the broken name
+--FILE--
+<?php
+$m = new MongoDB\Driver\Manager("mongodb://localhost:44444");
```

Comment:
It should never connect to anything as the exception should be thrown before attempting to connect to anything.